src/lunchselector/core.clj

```diff
@@ -1,7 +1,42 @@
 (ns lunchselector.core
-  (:gen-class))
+  (:require [ring.util.response :as res]
+            [bidi.ring :refer (make-handler)]
+            [ring.middleware.session :refer [wrap-session]]
+            [lunchselector.fetch :refer [get-categories]]
+            [cheshire.core :refer :all]))
 
-(defn -main
-  "I don't do a whole lot ... yet."
-  [& args]
-  (println "Hello, World!"))
+(defn homepage [request]
+  (res/response "<html>
+<head>
+<title>HomePage!!</title>
+</head>
+<body>
+<form action=\"article\" method=\"post\">
+Name: <input type=\"text\" name=\"name\"><br/>
+<input type=\"submit\" value=\"submit\">
+</form>"))
+
+(defn not-found [request]
+  {:status 404
+   :body "Not found!"})
```

Comment:
JFYI, I believe `ring.response` or some ring namespace has these utils. It's not too bad to define these here though.